pkg/hs/urbit-king/lib/Urbit/King/Main.hs

```diff
@@ -175,10 +175,10 @@ logStderr action = do
   logFunc <- view stderrLogFuncL
   runRIO logFunc action
 
-logSlogs :: HasStderrLogFunc e => RIO e (TVar (Text -> IO ()))
+logSlogs :: HasStderrLogFunc e => RIO e (TVar ((Atom, Tank) -> IO ()))
```

Comment:
Because a slog is a pair of `[priority valAsTank]`. The previous code rendered the tank at this part of the code and then stuck the rendered text into the logging function, which isn't correct. To actually implement this correctly, we need to thread the unrendered value to the console subsystem which can render it with the correct width.